README.md

```diff
@@ -53,7 +52,21 @@ class ExpensiveRenderer extends React.Component {
 }
 ```
 
-## Benchmarking
+## Do I Need `shouldComponentUpdate`?
```

Comment:
I'd change to `Do I Need this Library?` since sCU is a potentially smaller world than all of the needs for deep equality.

Reply:
Ah yes! So because this lib is also a general purpose I wanted some want to indicate that these words of caution are particular to sCU. Hmmmm....